### ISSUE TITLE
memory leak array and dictionary

### ISSUE THREAD

**Issue by kolisko** [SwiftyJSON/SwiftyJSON#601]
Hi, I have the following code:

``` swift
var json = JSON(response.result.value!) // response from Alamofire json call
var d = json["Data"].array // this cause memory leaks
d = nil
json = nil
```

Any idea how to fix the leak?

**Comment by nhatlee** [comment]
Any update?

**Comment by guidove** [comment]
Have the same problem. Found this thread on Alamofire github: https://github.com/Alamofire/Alamofire/issues/1200 
Seems they also believe it's a JSONSerialization leak (so I'm guessing SwiftyJSON leak??)

Any pointers as to how to solve or workaround this?

**Comment by wongzigii** [comment]
Maybe related to https://github.com/Alamofire/Alamofire/issues/1755

**Comment by wongzigii** [comment]
I didn't see this leak in the latest Xcode 8.2. It seems this issue has been resolved by Swift 3.0.2. Closing.

**Comment by Nehal-29** [comment]
getting memory leak on 
"JSONSerialization.jsonObject(with: data, options: []) as? [String: Any]"
